src/integrationTest/resources/application-intTest.yaml

```diff
@@ -47,6 +47,7 @@ darts:
       arm-username: some-username
       arm-password: some-password
       url: http://localhost:4551
+      update-metadata-path: /api/v3/UpdateMetadata
```

Comment:
Please can you remove this line - as it's already in `src/main/resources/application.yaml`